src/layers/core/grid-cell-layer/grid-cell-layer.js

```diff
@@ -66,15 +66,20 @@ export default class GridCellLayer extends Layer {
 
   getShaders() {
     return enable64bitSupport(this.props) ? {
-      vs: gridCellVertex64, fs: gridCellFragment, modules: ['fp64', 'project64', 'lighting']
+      vs: gridCellVertex64,
```

Comment:
I found some lines with multiple modules extending over 100 characters limit... that's why I break them into multiple lines...